File: Filza/FilzaBarang.cpp
#include <iostream.h>
int main()
{
	int barang[] = {100000, 750000, 150000, 200000, 75000};
	int mouse, keyboard, speaker, headset, cooling, totalbayar;
	cout << "1. Mouse Optik ("<<barang[0]<<")";
	cout << "2. Keyboard ("<<barang[1]<<")";
	cout << "3. Speaker ("<<barang[2]<<")";
	cout << "4. Headset ("<<barang[3]<<")";
	cout << "5. Cooling ("<<barang[4]<<")";
	cout << endl;

	cout << "Mau Beli Mouse Optik berapa buah? = ";
	cin >> mouse;

	cout << "Mau Beli Keyboard berapa buah? = ";
	cin >> keyboard;

	cout << "Mau Beli Speaker berapa buah? = ";
	cin >> speaker;

	cout << "Mau Beli Headset berapa buah? = ";
	cin >> headset;

	cout << "Mau Beli Cooling berapa buah? = ";
	cin >> cooling;

	totalbayar =( (barang[0]*mouse) + (barang[1]*keyboard) + (barang[2]*speaker) + (barang[3]*headset) + (barang[4]*cooling) );

	cout << "Total Bayar = "<<totalbayar;
}

File: Filza/FilzaBarang.java
import java.io.*;

public class FilzaBarang
{
	static int Larik[] = {100000, 750000, 150000, 200000, 75000};
	static int mouse, keyboard, speaker, headset, cooling, i, j, totalbayar;
	
	public static void main(String[] args)
	{
		System.out.println("1. Mouse Optik (" + Larik[0] + ")");
		System.out.println("2. Keyboard (" + Larik[1] + ")");
		System.out.println("3. Speaker (" + Larik[2] + ")");
		System.out.println("4. Headset (" + Larik[3] + ")");
		System.out.println("5. Cooling (" + Larik[4] + ")");

		System.out.println("");

		System.out.print("Mau Beli Mouse Optik berapa buah? = ");
		mouse = inputData();

		System.out.print("Mau Beli Keyboard berapa buah? = ");
		keyboard = inputData();

		System.out.print("Mau Beli Speaker berapa buah? = ");
		speaker = inputData();

		System.out.print("Mau Beli Headset berapa buah? = ");
		headset = inputData();

		System.out.print("Mau Beli Cooling berapa buah? = ");
		cooling = inputData();	

		totalbayar =( (Larik[0]*mouse) + (Larik[1]*keyboard) + (Larik[2]*speaker) + (Larik[3]*headset) + (Larik[4]*cooling) );

		System.out.println("");
		
		System.out.println("Total Bayar = " + totalbayar);
	}
	
	private static int inputData()
	{
		BufferedReader dataIn = new BufferedReader ( new InputStreamReader ( System.in));
		
		String angkaInput = null;
		try
		{
			angkaInput = dataIn.readLine();
		}
		catch ( IOException e )
		{
			e.printStackTrace();
		}
		
		int Data = Integer.valueOf(angkaInput).intValue();
		return Data;
	}
	
}